content/blog/2023/2023-09-25-introduction-to-kotest.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Introduction to Kotest"
+categories: ["Kotlin"]
+date: 2023-09-25 00:00:00 +1100 
+authors: [ezra]
+excerpt: "In this tutorial, we'll discuss Kotest which is a testing Framework  written in Kotlin."
+image: images/stock/0104-on-off-1200x628-branded.jpg
+url: introduction-to-Kotest
+---
+
+
+
+Kotest is simply a multi-platiform framework used for testing and it's written in Kotlin.In this tutorial, we shall cover the following sub-topics related to the Kotest framework: testing with Kotest, testing styles used, grouping Kotest tests with tags, the lifecycle hooks and assertions supported by Kotest.
+
+## Testing on JVM
+In this step, we're going to learn how we can test our Kotlin code using Kotest. Before writing the tests, we shall first have to add Kotest framework dependencies in our project. 
+
+Let's take a look at how we add the dependencies to our *pom.xml* file:
```

Comment:
Instead of using italics, please use `code` formatting for filenames, method names and variable names. Please check the whole article for instances of this.

Reply:
Note that you don't need to use `<code>` as you did in you last commit, but you can just use tick marks for code highlighting (`).